operator/v1/types_etcd.go

```diff
@@ -28,12 +28,33 @@ type Etcd struct {
 
 type EtcdSpec struct {
 	StaticPodOperatorSpec `json:",inline"`
+	// profile allows user to change the etcd tuning profile which configures
+	// the latency parameters for heartbeat interval and leader election timeouts
+	// allowing the cluster to tolerate longer round-trip-times between etcd members.
+	// Valid values are "Default" and "Slower".
+	// The current default is Default.
+	// +kubebuilder:validation:Optional
+	// +openshift:enable:FeatureSets=CustomNoUpgrade;TechPreviewNoUpgrade
+	// +optional
+	Profile ProfileValue `json:"profile,omitempty"`
 }
 
 type EtcdStatus struct {
 	StaticPodOperatorStatus `json:",inline"`
 }
 
+const (
+	// DefaultProfile sets Heartbeat interval: 100ms, Leader election timeout: 1000ms
```

Comment:
Will any of heartbeat interval/leader election timeout change in the future? I.e. can the default values for the Default profile change? E.g. based on some performance tuning through various OCP releases. The same question for the Slower profile.

Reply:
Are the values going to be identical for all providers? Or, is there a possibility the default values will differ per each provider?

Reply:
The mappings for each profile should be static. We'll add additional profiles if we need different values.

The values will be identical for all providers. If a certain provider needs a different starting profile, we can work to have that set for that provider by default.

For example, if we find that Azure, for whatever reason, needs to be on the Slower profile out of the box, we can add logic to set the initial profile to Slower.

Reply:
YOu should still describe intent and indicate that the values are subject to change without notice.